src/widgets/ComboBox.js

```diff
@@ -0,0 +1,194 @@
+import _ from 'lodash';
+import KeyCodes from 'keycodes-enum';
+import AccessibilityModule from '@curriculumassociates/createjs-accessibility';
+import SingleLineTextInput from './SingleLineTextInput';
+
+/**
+ * A combobox that allows the user to enter whatever they like and the listbox
+ * part is suggested values
+ */
+export default class ComboBox extends createjs.Container {
+  constructor(options, width, height, tabIndex) {
```

Comment:
nit- as a reader it's not immediately clear what is mandatory vs. optional for the constructor. maybe esdoc + defaults?